open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -11,10 +11,25 @@ servers.
 See the [Egeria Planning guide](https://github.com/odpi/egeria/tree/master/open-metadata-publication/website/planning-guide) to familiarise yourself with 
 the types of [Egeria OMAG Servers](../../admin-services/docs/concepts/omag-server.md); Presentation Server is one of those servers.  
 
+There are 2 types of capabilities exposed in the Presentation Server.
+- Integration Server capabilities
+- Governance Solution capabilities  
+
+### Integration Server capabilities
+
+- Type Explorer: allowing the use to explore the Egeria open types
+- Repository Explorer: allowing the user to explore Egeria instances
+- Server author: allowing the user to configure a server. This is work in progress.
+- Dino: allowing a user to see and work with Egeria platforms and servers operationally.      
+
+### Governance Solution capabilities  
+- Glossary Author: allowing a Glossary author use to create, update and delete Glossary content.       
 
 ## Setting up the downstream servers  
  To run the Presentation server, you need to have correctly [configured](../../admin-services/docs/user/configuring-an-omag-server.md) and be [operating](../../admin-services/docs/user/operating-omag-server.md) the downstream servers.
- For example for the Glossary Author capability, the Presentation server issues omvs calls to the [View server](../../admin-services/docs/concepts/view-server.md)
+ Be aware that in addition to any existing OMAG Servers, the Presentation Server requires that you have [configured the view services](../../admin-services/docs/user/configuring-the-view-services.md) on a View Server. 
+
+ For example for the Glossary Author capability, the Presentation server issues OMVS calls to the [View Server](../../admin-services/docs/concepts/view-server.md)
```

Comment:
Should expand OMVS the first time - i.e. .... "Open Metadata View Services (OMVS) calls" ...